ariadne/contrib/federation/utils.py

```diff
@@ -1,9 +1,15 @@
 # pylint: disable=cell-var-from-loop
 
-import re
 from inspect import isawaitable
 from typing import Any, List
+from typing import cast
+from typing import Tuple
 
+from graphql import DirectiveDefinitionNode
+from graphql import Node
+
```

Comment:
Unnecesary empty line, also please bunch imports from modules into single `import from (A, B, C)` instead of repeated lines.

Reply:
Will fix. Thanks for the callout